src/Uno.UWP/Data/Pdf/PdfDocument.Droid.cs

```diff
@@ -0,0 +1,81 @@
+#if __ANDROID__
```

Comment:
This file should be named ".android.cs", so that the globbing picks it up: 
https://github.com/unoplatform/uno/blob/a6db376e46afd9fb30dee317f08a2e50dc5310e4/src/PlatformItemGroups.props#L70

Reply:
sorry, I used ".Droid.cs" naming convention [from here](https://github.com/unoplatform/workshops/tree/master/uno-bootcamp/modules/07-Working-with-Uno#ui-implementations)

Reply:
I made a PR to fix the docs